site/src/_layouts/component.hbs

```diff
@@ -37,6 +37,14 @@ layout: default
 
     <nav class="col-sm-3 hidden-xs">
       <ul id="sidebar" class="nav nav-stacked fixed">
+        <li>
```

Comment:
Hmmm ... I'll have a check and see how that pans out - although I do want one of them to be `index.html`.